expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -55,14 +54,6 @@ public class ResourceExpansionServiceImpl implements ResourceExpansionService {
     private final RawContentRetriever authorizedUriRetriever;
     private final RawContentRetriever uriRetriever;
 
-    public ResourceExpansionServiceImpl(ResourceService resourceService,
-                                        TicketService ticketService) {
-        this.resourceService = resourceService;
-        this.ticketService = ticketService;
-        this.authorizedUriRetriever = new UriRetriever();
```

Comment:
Removing redundant constructor causing issues with coverage, while avoiding implicit generation of an UriRetriever that we cannot mock. The other constructor forces us to supply it as parameter.